Upstream-Status: Pending

Otherwise automake shows a lot of warnings about subdir-objects and then fails

https://launchpad.net/ubuntu/trusty/+source/presage/0.8.8-1ubuntu3

Description: Disable automake -Werror.
Author: Dimitri John Ledkov <xnox@ubuntu.com>

diff -uNr presage-0.8.9.orig/configure.ac presage-0.8.9/configure.ac
--- presage-0.8.9.orig/configure.ac	2014-01-15 23:25:54.963166257 +0100
+++ presage-0.8.9/configure.ac	2014-01-15 23:27:23.287162792 +0100
@@ -23,7 +23,7 @@
 
 
 AC_INIT([presage],[0.8.9],[matteo.vescovi@yahoo.co.uk])
-AM_INIT_AUTOMAKE([-Wall -Werror])
+AM_INIT_AUTOMAKE([-Wall])
 
 AC_CONFIG_SRCDIR([src/lib/presage.cpp])
 AC_CONFIG_HEADERS([config.h:config.hin])
